kubernetes/sample/example-configmap.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: caddy-global-options
+  namespace: caddy-system
+data:
+  acme-ca: https://acme-staging-v02.api.letsencrypt.org/directory
```

I renamed `acme-ca` to `acmeCA` and forgot this sample.

```suggestion
  acmeCA: https://acme-staging-v02.api.letsencrypt.org/directory
```